src/runner/tray_icon.cpp

```diff
@@ -192,7 +192,9 @@ void start_tray_icon()
         tray_icon_data.hWnd = hwnd;
         tray_icon_data.uID = id_tray_icon;
         tray_icon_data.uCallbackMessage = wm_icon_notify;
-        wcscpy_s(tray_icon_data.szTip, sizeof(tray_icon_data.szTip) / sizeof(WCHAR), L"PowerToys");
+        std::wstring about_msg_pt_version = L"PowerToys\n" + get_product_version(); // get the version: PowerToys\nVersion
+        wcscpy_s(tray_icon_data.szTip, sizeof(tray_icon_data.szTip) / sizeof(WCHAR), _wcsdup(about_msg_pt_version.c_str())); // convert type std::wstring to wchar_t* and show
+        
```

Comment:
```suggestion
        std::wstring about_msg_pt_version = L"PowerToys\n" + get_product_version();
        wcscpy_s(tray_icon_data.szTip, sizeof(tray_icon_data.szTip) / sizeof(WCHAR), about_msg_pt_version.c_str());        
```
1. no need for these comments, they do not provide any information
2. no need for `_wcsdup`. it also leaks memory